### ISSUE TITLE
Gumball example - CORS issue

### ISSUE THREAD

**Issue by croquelois** [radixdlt/babylon-alphanet#30]
Running the Gumball example, I got a CORS error:
![image](https://user-images.githubusercontent.com/1538314/196277058-72e55ab7-4a17-4f8b-b858-133d9eaf163a.png)

**Comment by cbisaillon** [comment]
Is this only when it returns an error as in this issue #31 ?

**Comment by croquelois** [comment]
yes, I think it's because there is an error returned both issue can be merged
